### PR TITLE
feat(python): emit suggestion for how to replace map_elements sigmoid function with expressions

### PR DESCRIPTION
**Pull request by MarcoGorelli**
towards https://github.com/pola-rs/polars/issues/9968

Saw this one on Kaggle...uurgghh

The change here looks more complicated that it is...going to try to do a refactor first (edit: done in #13348)

### CODE REVIEW

**Comment by ritchie46** [comment]
> Saw this one on Kaggle...uurgghh

Being vigilante. :smile:

File: py-polars/polars/utils/udfs.py [hunk 1]
```diff
@@ -192,6 +192,14 @@ class OpNames:
         "function_name": [{"strptime"}],
     },
 ]
+_UNARY_OPNAME_POSSIBILITIES: list[list[AbstractSet[str]]] = [[set(OpNames.UNARY)], []]
```
Comment (alexander-beedie) [inline py-polars/polars/utils/udfs.py:195]: Minor nitpick; this doesn't actually seem to expand the unary operator possibilities, it just allows the existing unary ops to be optionally present when matching? In which case I'd just bake it in directly like so...
```python
_FUNCTION_KINDS = [
    {**kind, "argument_1_unary_opname": unary}
    for kind in _FUNCTION_KINDS
    for unary in [[set(OpNames.UNARY)], []]
]
```

Comment (MarcoGorelli) [reply py-polars/polars/utils/udfs.py:195]: thanks! i only did this to appease mypy, maybe a `type ignore` is warranted here

Comment (alexander-beedie) [reply py-polars/polars/utils/udfs.py]: Works for me :)

**Review by alexander-beedie — APPROVED** [review]

**Pull request merged**